File: AmpedBiz/AmpedBiz.Web/src/common/models/branch.ts
import { Address } from './address';
import { Contact } from './contact';
import { Tenant } from "./tenant";

export type Branch = {
  id?: string;
  name?: string;
  tenant?: Tenant;
  description?: string;
  taxpayerIdentificationNumber?: string;
  contact?: Contact;
  address?: Address;
}

export type BranchPageItem = {
  id?: string;
  name?: string;
  description?: string;
  address?: Address;
}

File: AmpedBiz/AmpedBiz.Service/Dto/Branch.cs
using System;

namespace AmpedBiz.Service.Dto
{
    public class Branch
    {
        public Guid Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string TaxpayerIdentificationNumber { get; set; }

        public Contact Contact { get; set; }

        public Address Address { get; set; }
    }

    public class BranchPageItem
    {
        public string Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }
    }
}
